Sources/WebDriver/Requests.swift

```diff
@@ -541,6 +541,29 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidlocation
+    public enum SessionLocation {
+        public struct Post: Request {
+            public var session: String
+            public var latitude: Double
```

Comment:
`public var location: Location` to avoid redundancy

Reply:
I think the changes I just pushed up address this but if not please let me know.

Reply:
We're now storing (latitude, longitude, altitude) redundantly with location. We should remove the former 3, so `body` becomes `public var body: Location { location }`.